package mio68.lab.tryit.puzzlers;

import java.io.IOException;

public class Puzzle37_Arcane1 {
    public static void main(String[] args) {
//        Compilation Error
//        java: exception java.io.IOException is never thrown in body of corresponding try statement
//        try {
//            System.out.println("Hello World!");
//        } catch (IOException e) {
//            System.out.println("I've never seen println fail!");
//        }
    }
}
